app/src/main/java/com/cornellappdev/volume/data/models/Publication.kt

```diff
@@ -25,6 +25,7 @@ data class Publication(
     val slug: String,
     val shoutouts: Double,
     val websiteURL: String,
+    val contentTypes: List<String>,
```

Comment:
I think this would be better as a list of ENUMS than raw strings so it scales better from the source of truth;

should be

``` 
    val contentTypes: List<ContentType>,
    val websiteURL: String,
    val numArticles: Double,
    val mostRecentArticle: Article? = null,
    val socials: List<Social>
)

enum class ContentType {
    MAGAZINES, ARTICLES
}
```

Reply:
Implemented these changes on emily-content-type (so that they can eventually be merged into main), then incorporated those changes into this branch. This way we can have the enum change in main next semester, and keep the filtering only in this branch.